client/components/searchbar.jsx

```diff
@@ -4,20 +4,30 @@ export default class Searchbar extends React.Component {
   constructor(props) {
     super(props);
     this.state = ({
-      search: ''
+      value: ''
     });
-    this.handleChange = this.handleChange.bind(this);
+
+    this.getSearchTerm = this.getSearchTerm.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    const search = this.state.value;
+
+    this.props.search(search);
+
+    this.setState({ value: '' });
   }
 
-  handleChange(event) {
-    this.setState({ search: event.target.value });
+  getSearchTerm(event) {
+    this.setState({ value: event.target.value });
   }
 
   render() {
     return (
       <div className="search-container">
-        <input className="searchbar" value={this.state.search} onChange={this.handleChange} placeholder="Search for artist, sports team, or venue"></input>
-        <button className="search-btn">SEARCH</button>
+        <input className="searchbar" value={this.state.value} onChange={this.getSearchTerm} placeholder="Search for artist, sports team, or venue"></input>
+        <a href="#results" className="search-btn" onClick={this.handleClick}>SEARCH</a>
```

Comment:
This might have been part of what you were asking about earlier. This should just be a button, you don't want to combine an action with an anchor tag. You can programmatically change the location in App in the getSearchResults method. See next comment...